src/routes/dataset.ts

```diff
@@ -251,3 +228,32 @@ router.get('/:dataset_id/list-files', loadDataset(withDraftForCube), listAllFile
 // PATCH /dataset/:dataset_id/group
 // Updates the user group for the dataset
 router.patch('/:dataset_id/group', jsonParser, loadDataset(), updateDatasetGroup);
+
+router.use(
+  '/:dataset_id/revision',
+  rateLimiter,
+  passport.authenticate('jwt', { session: false }),
+  loadDataset({
+    dimensions: { metadata: true, lookupTable: true },
+    factTable: true,
+    measure: { measureTable: true, metadata: true },
+    revisions: { dataTable: { dataTableDescriptions: true } }
+  }),
+  revisionRouter
+);
+
+router.use(
+  '/:dataset_id/dimension',
+  rateLimiter,
+  passport.authenticate('jwt', { session: false }),
+  loadDataset(withDraftForCube),
+  dimensionRouter
+);
+
+router.use(
+  '/:dataset_id/measure',
+  rateLimiter,
+  passport.authenticate('jwt', { session: false }),
+  loadDataset(),
```

Comment:
This is the main perf improvement in this PR - previously it was 
```loadDataset(withDraftForCube),```

which hydrates: 
```{
  factTable: true,
  draftRevision: { dataTable: { dataTableDescriptions: true } },
  dimensions: { metadata: true, lookupTable: true },
  measure: { metadata: true, measureTable: true, lookupTable: true },
  revisions: { dataTable: { dataTableDescriptions: true } }
}
```
... a lot of which we don't actually need in most of the measures routes.